index.js

```diff
@@ -43,6 +43,9 @@ app.post(
   userRegister
 );
 
+// login
+app.post("/user/login", login);
```

Comment:
add a validator here for checking email and password. they should not be null @amrit56789